compiler/src/dotty/tools/dotc/typer/Migrations.scala

```diff
@@ -160,9 +160,7 @@ trait Migrations:
   end implicitParams
 
   private def checkParentheses(tree: Tree, pt: FunProto)(using Context): Boolean =
-    ctx.source.content
-      .slice(tree.span.end, pt.args.head.span.start)
-      .exists(_ == '(')
+    pt.args.head.span.exists && ctx.source.content.slice(tree.span.end, pt.args.head.span.start).exists(_ == '(')
```

Comment:
style proposal
```
pt.etc.exists
&& ctx.etc.content
  .slice()
  .exists(p)
```
The most informative part is the predicate about the paren, which ought to align left instead of being hidden way off to the right margin.

Also consider the complexity of the slice args; burying it in a long expr makes it hard to verify visually.

I used to a one-liner person but Scala 3 has made me evolve. I use leading infix generously.

I still put low-info bits to the right, such as `def f(tree: Tree) = tree match`.

This is just IMHO but I wanted to take the opportunity to do a good deed.

Reply:
In my defense, I think this is style I picked up from the scala 3 code base, so I have more confidence that it is beneficial than if it were something I made up myself. In any case, I appreciate the you did not, having listened, ignore me outright.